app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -31,6 +31,7 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
 
     private var selectedAccountUuid: String? = null
     private var selectedFolderId: Long? = null
+    private var selectedFolderDisplayName: String? = null
```

Comment:
This property has the same "lifetime" as `selectedFolderId`, i.e. it should be set to `null` whenever `selectedFolderId` is set to `null`.